.github/actions/performance-tests/art.yaml

```
@@ -524,13 +524,14 @@
+              as: response
           ensure:
-            - statusCode: 200
-            - statusCode: 201
-          assert:
-            - subject: response.error
-              equals: null
+            - statusCode:
+                - 200
+                - 201
+          expect:
+            - notHasProperty: "{{ response.error }}"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)